app/src/main/java/org/glucosio/android/tools/LocaleHelper.java

```diff
@@ -54,35 +54,21 @@ public void updateLanguage(@NonNull final Context context, @NonNull final String
 
     @NonNull
     public List<String> getLocalesWithTranslation(final Resources resources) {
-        String[] languages = BuildConfig.TRANSLATION_ARRAY;
+        String[] languageList = BuildConfig.TRANSLATION_ARRAY;
         Set<String> availableLanguagesSet = new HashSet<>();
-        // We always support english
-        availableLanguagesSet.add("en");
-
-        // Get english string to confront
-        // I know, it's a weird workaround
-        // Sorry :/
-        String englishString = "Automatic backup";
 
-        for (String localString : languages) {
-            // For each locale, check if we have translations
-            Configuration conf = resources.getConfiguration();
-            Locale savedLocale = conf.locale;
-            conf.locale = getLocale(localString);
-            resources.updateConfiguration(conf, null);
+        String[] translatedLanguages = resources.getStringArray(R.array.available_languages);
```

Comment:
you should aware that if there is 1% translated language and it translated only "Automate Backup", It will be added to supporting language. It should be removed.